### ISSUE TITLE
Clojars artifact

### ISSUE THREAD

**Issue by yogthos** [nextjournal/clojure-mode#15]
Any chance of an artifact getting published to Clojars to make it easier to use with Leiningen based projects? :)

**Comment by mk** [comment]
Hi @yogthos, we've been avoiding publishing it as a jar to signal the alpha / subject to change status of the api. Can try to package things up if you understand that changes might still happen.

**Comment by yogthos** [comment]
Yeah that would work for me, and tagging the release version as alpha would probably be reasonable to indicate that it's subject to change. :+1:

**Comment by yogthos** [comment]
in the meantime [lein-tools-deps](https://github.com/RickMoynihan/lein-tools-deps) plugin will work by adding the following in `project.clj`

```clojure
:plugins [[lein-tools-deps "0.4.5"]]
:middleware [lein-tools-deps.plugin/resolve-dependencies-with-deps-edn]
:lein-tools-deps/config {:config-files [:install :user :project]}
```
and a `deps.edn`

```clojure
{:deps {nextjournal.clojure-mode {:git/url "https://github.com/nextjournal/clojure-mode"
                                  :sha "fcf51d3dbc1ad192d261a5417cff03c3f466dfa0"}}
 }
```
